modules/ROOT/pages/policies-policy-types.adoc

```diff
@@ -0,0 +1,49 @@
+= Policy Types
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies are available in different options, such as, Default, Custom, and Automated. Each type of the policy differs by its scope, management, and usability.
+
+== Default Policies
+
+Mulesoft provides several policies that are ready-to-use and cover different areas, such as authentication, security management, threat protection, and tokenization. You can apply any of these policies using the UI to any of your API endpoints. You can also use an API to apply policies.
+
+After you apply a policy, it is managed by the API Manager for its complete lifecycle. You can apply a Default policy to a specific API. To automatically apply a policy to to all APIs in your environment, you must configure it as  a policy.
```

Comment:
```suggestion
After you apply a policy, it is managed for its complete lifecycle by API Manager. You can apply a default policy to a specific API. To automatically apply a policy to all APIs in your environment, you must configure it as a policy.
```
The last sentence does not make sense. What kind of policy? Did you mean "automatic"?

Reply:
Agree, it should be "you must configure it as an automated policy"

Reply:
Done! Thanks!